### ISSUE TITLE
Document zoom_image() operation order

### ISSUE THREAD

**Issue by AnderBiguri** [SyneRBI/SIRF#813]
If you zoom and offset in the same call to `zoom_image()` the amount of offset changes by some unknown ration. 

I suppose this is because operation order?

e.g:

```
img.get_geometrical_info().print_info()
img=img.zoom_image(offsets_in_mm=(-10,0.0,0.0),zooms=(2.80005/2.76148, 1., 1.))
img.get_geometrical_info().print_info()
```
```
Offset: (-350, -350, 21.52)
Spacing: (1.36719, 1.36719, 2.80005)
Size: (512, 512, 411)
Dir mat: 
1, 0, 0
0, 1, 0
0, 0, -1

Offset: (-350, -350, 23.6132)
Spacing: (1.36719, 1.36719, 2.76148)
Size: (512, 512, 411)
Dir mat: 
1, 0, 0
0, 1, 0
0, 0, -1
```

the `zoom` is correct, but the offset is now `+2.0932` instead of `-10`. What is happening here? I can't figure it out, but if there is a logic behind this, then it should possibly be documented.

**Comment by KrisThielemans** [comment]
This is a STIR difficulty. Francesca struggled with it as well and so did I. I put some of that in https://github.com/SyneRBI/SIRF-Contribs/tree/master/src/Python/sirf/contrib/brainweb-utilities.

The convention is really rather hard, and I'd rather avoid this signature and use a template #696 

Depending on what you need, it might be easier to use `sirf.Reg` for resampling (`zoom_image` can use count-preserving interpolation, which niftyreg can't do, but does that matter for you?)

**Comment by AnderBiguri** [comment]
Agreed, the best way of doing this is via template. But its not supported yet! There should be either documentation to current code or a different code, but otherwise all this is in limbo. 

I guess we should definitely close this issue if #696 gets solved by overwriting the current usage.
